home-exercises-05-2022/src/io/codelex/arithmetic/practice/SumAverageRunningInt.java

```diff
@@ -0,0 +1,18 @@
+package io.codelex.arithmetic.practice;
+
+public class SumAverageRunningInt {
+    public static void main(String[] args) {
+        int sum = 0;
+        int lowerBound = 1;
+        int upperBound = 100;
+        int count = 0;
+
+        for (int number = lowerBound; number <= upperBound; ++number) {
+            sum += number;
+            count++;
```

Comment:
Šo count varētu aprēķināt matemātiski (upperBound - lowerBound) nevis tā teikt skaitīt :)

Reply:
Kad es ielieku (upperBound - lowerBound) man sanāk average 51.01010101. Bet ja es izmantoju count++ man sanāk 50.5.  Jo ja es skaitu ar count,  tad man sanāk 100 skaitļi no 1 līdz 100. Bet ja (upperBound - lowerBound), tad 99 skaitļi.  Es kautko daru nepareizi?

Reply:
Tur matemātiski vienkārši +1 būtu jāpieskaita, bet nu likšu approve.